FastSurferCNN/data_loader/data_utils.py

```diff
@@ -53,8 +53,9 @@ def load_and_conform_image(img_filename, interpol=1, logger=LOGGER, conform_min
     :return: nibabel.MGHImage orig: conformed image
     """
     orig = nib.load(img_filename)
-
-    if not is_conform(orig, conform_min=conform_min):
+    # is_conform and conform accept numeric values and the string 'min' instead of the bool value
+    _conform_vox_size = 'min' if conform_min else 1.
+    if not is_conform(orig, conform_vox_size=_conform_vox_size):
```

Comment:
Why is it not possible to conform to a given size in this function? Everywhere else this is possible, only here it is min or 1

Reply:
I just changed the call interface to work as before.
`load_and_conform_image()` is actually a stub as it is not called anywhere else as far as I can see.

Reply:
Yeah, but if someone is using it in the future, would it not make sense to either remove the function or adopt it to work in the same way (i.e. pass the voxsize directly to the conform call)?